ibverbs-sys/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "ibverbs-sys"
+version = "0.1.0"
```

Comment:
Wasn't sure whether to use the same version number as `ibverbs` or to start counting from scratch so they're "separate". Up to you!

Reply:
I think keeping them separate makes a lot of sense — that's almost certainly what we want anyway, so that we can cut releases independently for the two.